dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -320,6 +320,11 @@ object WavesDexCli extends ScoptImplicits {
               .valueName("<raw-string>")
               .required()
               .action((x, s) => s.copy(configPath = x)),
+            opt[String]("api-key")
```

Comment:
I think the key should be written by user via readSecretFromStdIn

Reply:
Agree. We need to read this secret carefully and prevent it from leaking to the bash history.

We have same parameter in `CreateApiKey`, but it covers another case:
* An admin does this once;
* When an admin should see the input and not make a mistake.

Reply:
resolved